Sources/Testing/Running/Configuration.swift

```diff
@@ -180,3 +168,36 @@ public struct Configuration: Sendable {
   @_spi(ExperimentalParameterizedTesting)
   public var testCaseFilter: TestCaseFilter = { _, _ in true }
 }
+
+// MARK: - Test filter factory functions
+
+/// Filter tests to run to those specified via a set of test IDs.
```

Comment:
Can we update the wording of these two factory functions? Now, they don't _perform_ the filtering, but only generate a filter

Reply:
Oh, sure.